src/css/structure/jquery-ui.css

```diff
@@ -1126,7 +1126,8 @@ a.ui-button:focus {
     height : 16px;
 }
 
-.ui-icon,
+
+/*.ui-icon,
```

Comment:
I think the change should be made to the `jquery-ui-overrides.less` instead of touching the jQuery UI theme directly. If the jQuery UI css is updated, the commented out parts will come back.

Reply:
Sure this makes sense.